### DISCUSSION TITLE
您好，又来叨扰您了>< 请问如何把标题的Problem Chosen下的ABCDEF和Team Control Number下的数字改成红色？谢谢！麻烦您了！

### DISCUSSION THREAD

**Discussion by R-NoXX** [xjtu-blacksmith/easymcm#11] [Q&A] [answered]
谢谢！抱歉再次打扰了><

**Comment by Simon9Zhou** [comment, marked as answer]
标题红色跟黑色我看作者的介绍说是，美赛会把需要小组自己修改的部分标成红色作为提醒，小组完成之后改成黑色提交就可以，所以黑色没问题
但是如果一定要改

我自己尝试了一下在easymcm.sty文件中加几句代码：

% COMAP 要求的 Summary Sheet 标题（2020）
% 注意每年比赛时有可能有小调整
% 请以官网发布的样式为准自行做小的修改！
在以上这几句话后面那一段代码里面：

\LARGE\@problem
这一行替换成
\textcolor{red}{\Large\@problem\\[1.3pc]}  %_pc改变的是标题离下方黑线的距离

同时
\LARGE\MCM@control\\[1.8pc]
这一行替换成
\textcolor{red}{\Large\MCM@control\\[1.3pc]}  %_pc改变的是标题离下方黑线的距离
编译两次即可改变标题颜色

但是我自己弄过程中有可能报错，如果报错无法解决，可以替换成原本的代码使用黑色

**Reply by R-NoXX** [reply]
好的！非常感谢！
